CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-## Unreleased
+## 0.5.1
```

Comment:
You haven't added a changelog entry for this change, this also has the added benefit of setting up the ##unreleased section

Reply:
https://www.youtube.com/watch?v=1p-DA0zvuQ8